src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_stacks/adf_default_pipeline.py

```diff
@@ -46,8 +46,8 @@ def generate_adf_default_pipeline(scope: core.Stack, stack_input):
 
     pipeline_triggers = stack_input["input"].get("triggers", {}).get("triggered_by", None)
     if pipeline_triggers:
-        for k, v in pipeline_triggers.items():
-            _pipeline.add_pipeline_trigger((k,v))
+        for _trigger_type, _trigger_config in pipeline_triggers.items():
+            _pipeline.add_pipeline_trigger(trigger_type=_trigger_type, trigger_config=_trigger_config)
```

Comment:
In ADF, the use of underscored variables is not very well structured unfortunately.
In this case, there is no need to add the underscore. It would be required only when the variables are instance/class variables that need protection.

```suggestion
        for trigger_type, trigger_config in pipeline_triggers.items():
            _pipeline.add_pipeline_trigger(trigger_type=trigger_type, trigger_config=trigger_config)
```